### PR TITLE
Adds email communications guide to docs, exposes other hidden guides in nav bar

### LINKED ISSUE #1064: Add docs for email config
### Description

We need docs for email config api usage. API implementation done here- https://github.com/ethyca/fidesops/pull/1059

### PR DESCRIPTION
**Pull request by eastandwestwind**
# Purpose
Adds guide on email communications and setting up an email config/secrets to docs.

Also exposes previously hidden guides in the docs nav bar.

# Changes
- Update docs

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [x] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #1064

### CODE REVIEW

**Comment by eastandwestwind** [comment]
screencapture-0-0-0-0-8000-fidesops-guides-email-communications-2022-08-31-11_29_30

File: docs/fidesops/mkdocs.yml [hunk 1]
```diff
@@ -18,10 +18,18 @@ nav:
           - Managing Datastores: ui/datastores.md
           - User Management: ui/user_management.md
   - How-To Guides:
+      - Application Configuration Reference: guides/configuration_reference.md
+      - View Available Connection Types: guides/connection_types.md
+      - Connect to SQL and NoSQL Databases: guides/database_connectors.md
+      - Configure Datasets: guides/datasets.md
+      - Configure Email Communications: guides/email_communications.md
+      - Authenticate with Oauth: guides/oauth.md
```
Comment (conceptualshark) [inline docs/fidesops/mkdocs.yml:26]: A section of these docs were reordered to lines 7-12 and don't need to be listed twice, they just need to be rehomed in the project directory. 

It looks like the Connection Type guide is new? 🤔

File: docs/fidesops/mkdocs.yml [hunk 2]
```diff
@@ -18,10 +18,18 @@ nav:
...
       - Annotate Complex Fields: guides/complex_fields.md
       - Configure Data Masking: guides/masking_strategies.md
       - Configure Storage Destinations: guides/storage.md
+      - Configure Policies: guides/policies.md
       - Configure Policy Webhooks: guides/policy_webhooks.md
+      - Execute Privacy Requests: guides/privacy_requests.md
```
Comment (conceptualshark) [inline docs/fidesops/mkdocs.yml:32]: Same as above, I believe both these docs got moved up to the Intro section.

**Review by conceptualshark — CHANGES_REQUESTED** [review]

File: docs/fidesops/mkdocs.yml (hunk 1 above)
Comment (eastandwestwind) [reply docs/fidesops/mkdocs.yml:26]: ohhh I see, yup, looks like just the connection type guide wasn't listed in the nav before. It's relatively new, added July 6th 2022. I'll go ahead and remove the others from this `guides` section.

**Comment by conceptualshark** [comment]
I took a quick pass at cleaning some of this up to match some of the updated/incoming guides - let me know if it's what you had in mind!

**Comment by eastandwestwind** [comment]
@conceptualshark your updates look so much better, thanks for the help on this!

Back over to you

**Review by conceptualshark — APPROVED** [review]

**Pull request merged**